composer.json

```diff
@@ -107,7 +107,10 @@
     "pest:fast": [
       "@pest --parallel"
     ],
-    "phpcs": "@php vendor/bin/php-cs-fixer fix --dry-run --diff",
+    "phpcs": [
+      "@putenv PHP_CS_FIXER_IGNORE_ENV=1",
```

Comment:
Temporary workaround for PHPCS to run in the pre-release runtime environment successfully.